apps/src/templates/PaneHeader.jsx

```diff
@@ -82,7 +82,7 @@ var styles = {
  * A purple pane header that can have be focused (purple), unfocused (light purple)
  * or read only (charcoal).
  */
-var PaneHeader = React.createClass({
+const PaneHeader = Radium(React.createClass({
```

Comment:
Is there a benefit to wrapping each of these with Radium?

Other than automatic browser prefixing.